### PR TITLE
WIP: qubesos_3mdeb_minisummit: initial commit

### PR DESCRIPTION
**Pull request by pietrushnic**
Signed-off-by: Piotr Król <piotr.krol@3mdeb.com>

### CODE REVIEW

File: blog/content/post/2019-07-02-qubes-os-and-3mdeb-minisummit.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+* OE/Yocto not verify hashes or does it in doubtful way.
+* We can't freeze dom0, we need flexibility in extending it additional software
+  through package manager - as Marek mentioned this statement is valid until
+  GUI is in dom0. Goal is definitely to get rid of GUI from dom0.
+* We should treat big distros like Debian and Fedora as role models in package
+  management, they already use lots of good practice that are not followed by
+  OE/Yocto, because of that 3mdeb should understand better how Debian/Fedora
+  does package management and help reflect best practice in OE/Yocto to improve
+  probability of using it as dom0 build system.
```
Comment (marmarek) [inline blog/content/post/2019-07-02-qubes-os-and-3mdeb-minisummit.md]: Very good analysis about package managers and updates is here: https://theupdateframework.github.io/
Especially this paper: https://theupdateframework.github.io/papers/package-management-security-tr08-02.pdf?raw=true

File: blog/content/post/2019-07-02-qubes-os-and-3mdeb-minisummit.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+# TPM2.0
+
+It looks that TPM2.0 is not supported in Qubes OS mostly because nobody had
+time to take a look at it. From 3mdeb it would be ideal contribution especially
+if can be combined with open-source firmware based hardware like
+[Librebox](https://shop.3mdeb.com/product/librebox/).
+
+There are various other ways how TPM2.0 can be leveraged in Qubes OS e.g. PKCS #11, VPN.
+
+# fwdupd
```
Comment (artur-rs) [inline blog/content/post/2019-07-02-qubes-os-and-3mdeb-minisummit.md]: `fwupd`?

**Comment by pietrushnic** [comment]
@marmarek I added all references and some comments also fixed some small issues that I found. We wait for your approval to publish that blog post.

File: blog/content/post/2019-07-02-qubes-os-and-3mdeb-minisummit.md [hunk 3]
```diff
@@ -68,6 +70,8 @@ As mentioned on slides linked above we discussed also:
   little bit for reasonably performing ARM systems. There is also some hope in
   OpenPOWER and RISC-V, but ports to those architectures would require
   reasonably market demand for which we don't see potential right now.
+  Interestingly there is 2BTC bounty for Power Architecture support in Qubes OS
+  you can read bout it [here](https://github.com/QubesOS/qubes-issues/issues/4318).
```
Comment (marmarek) [inline blog/content/post/2019-07-02-qubes-os-and-3mdeb-minisummit.md]: about

**Comment by pietrushnic** [comment]
@marmarek executed spellchecker one more time on the whole text. Typos fixed.

**Pull request merged**